.github/workflows/collection.yml

```diff
@@ -0,0 +1,47 @@
+name: Build Ansible Collection
```

Comment:
This file should not be added back, bad merge.

Reply:
Fixed.